src/ui/server/controllers/insurance/account/create/confirm-email/index.ts

```diff
@@ -14,10 +14,16 @@ export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.CREATE.CONFIRM_EMAIL
  * @param {Express.Response} Express response
  * @returns {Express.Response.render} Confirm email page
  */
-export const get = (req: Request, res: Response) =>
-  res.render(TEMPLATE, {
+export const get = (req: Request, res: Response) => {
+  const email = req.session.emailAddressToConfirm;
```

Comment:
can we destructure session since being used more than once

Reply:
I've added destructuring for this part, but we can't destructure in the second line because it's a delete call, needs to reference it without destructuring.